unit_py/test_simple_pendulum.py

```diff
@@ -118,6 +118,68 @@ def dynamics(t, x):
         # Compare the numerical and numerical integration of analytical model using scipy
         self.assertTrue(np.allclose(x_jiminy, x_rk_python, atol=TOLERANCE))
 
+    def test_imu_sensor(self):
+        """
+        @brief   Test IMU sensor on pendulum motion.
+
+        @details Since we don't have a simple analytical expression for the solution
```

Comment:
Why not to use the solution of the simulation done by jiminy itself ? We are expected the IMU data to be consistent with the solution of Jiminy, not the one computed using an external solver. In my opinion your current implementation is checking at the same time that the solution of the simulation is the right one, and that the sensor IMU data are valid, not just that the IMU data are valid.

Reply:
Two reasons:

 A - I like a standalone test better. Though from a software engineer point of view it's redundant (and thus harder to maintain), we can look at the test as a whole and say 'I validated that an IMU mounted on a pendulum gives the signal one would expect from an IMU on a pendulum).

 B - jiminy output log does not feature the acceleration - that way I'm indirectly checking the acceleration computed by jiminy. Though the test could of course be rewritten to compute the expected IMU signal at each time step, using the internal `engine.stepper.dxdt` variable.

Reply:
No problem! Could you mention that in the test description as a reminder ?